extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -16,19 +16,25 @@ const DirectoryLink = () => {
   );
 };
 
+export enum WarningMessageVariant {
+  default = "",
+  highAlert = "high-alert",
+  warning = "warning",
+}
+
```

Comment:
trying to get this API closer to the one used in SDS to make transitioning back to SDS easier in the future